bin/deploy

```diff
@@ -36,5 +36,14 @@ aws s3 sync \
 
 echo "Deployed styleguide to http://rcl.policygenius.com/$VERSION/index.html"
 
+echo '+++ Deploying styleguide latest to S3'
+aws s3 sync \
+  ./styleguide s3://rcl-pg/latest/ \
```

Comment:
@j-clark The cache thing is a one-liner in the deploy script, I believe. #2.. I'm not sure. But would like to discuss further with you offline